ui/widgets/autocomplete.js

```diff
@@ -539,9 +542,10 @@ $.widget( "ui.autocomplete", {
 		// Size and position menu
 		ul.show();
 		this._resizeMenu();
+		var pos = this.isRtl() ? { my: "right top", at: "right bottom", collision: "none" } : this.options.position;
```

Comment:
a better parallell would be position absolute elements which are NOT mirrored by default. Browsers will never change
```css
position: absolute;
top:0;
left:0;
```
to
```css
position: absolute;
top:0;
right:0;
```